src/components/DashboardComponents/DashboardMain.js

```diff
@@ -0,0 +1,37 @@
+import { Box, Heading, Flex, Button } from "@chakra-ui/react";
+import dashboardImage from "../../headerImg.jpg"
+
+const DashboardMain = () => {
+    const auth = JSON.parse(localStorage.getItem("auth"))
```

Comment:
Please use the auth hook to access the auth (user) data. Check how this is accessed in the other components:

```
const { user } = useAuth()
```

This way the logic for accessing the user (auth) data (taken from the local storage) lives in the useAuth hook only.

Reply:
Done